namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ThriftNamerInterface.scala

```diff
@@ -362,16 +362,23 @@ object ThriftNamerInterface {
   case class Capacity(
     bindingCacheActive: Int,
     bindingCacheInactive: Int,
+    bindingCacheInactiveTTLSecs: Int,
     addrCacheActive: Int,
-    addrCacheInactive: Int
+    addrCacheInactive: Int,
+    addrCacheInactiveTTLSecs: Int
   )
 
   object Capacity {
+
+    val Week: Int = 60 * 60 * 24 * 7
```

Comment:
Out of curiosity, what is the motivation for having the binding cache TTL set to a week?

Reply:
I think `10.minutes` is probably a fairly reasonable default.  (if you import com.twitter.conversions.time._ then you can use `10.minutes`, `7.days`, etc.)